lib/src/widgets/desktop_nav_bar.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:cake_wallet/di.dart';
+import 'package:cake_wallet/store/settings_store.dart';
+import 'package:cake_wallet/themes/theme_base.dart';
+import 'package:flutter/material.dart';
+
+class DesktopNavbar extends StatelessWidget implements PreferredSizeWidget {
```

Comment:
let's rename this to something more specific since this design is only for dashboard navbar, maybe something like `DesktopDashboardNavbar` and also rename the file, plus we can move this to the `src/screens/dashboard/desktop_widgets`